.github/workflows/container.yml

```
@@ -13,87 +17,48 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
```

Required for is-latest-tag

```suggestion
        uses: actions/checkout@v3
        with:
          fetch-depth: 0
```